probability.py

```diff
@@ -549,7 +549,8 @@ def forward(HMM, fv, ev):
                             scalar_vector_product(fv[1], HMM.transition_model[1]))
     sensor_dist = HMM.sensor_dist(ev)
 
-    return([float("{0:.4f}".format(i)) for i in normalize(element_wise_product(sensor_dist, prediction))])
```

Comment:
Not your fault, @anuj1995 , but I think this line should be changed to just return the normalized results; then somewhere else we can have a function to truncate to 4 decimal places, if we find that more convenient to look at.